resources/css/articles.css

```diff
@@ -1,3 +1,34 @@
 .article p {
     @apply mb-4;
-}
\ No newline at end of file
+}
+
+.article h1,
+.article h2,
+.article h3 {
+    @apply mb-1;
+}
+
+.article h1 {
+    @apply text-3xl font-bold;
+}
+
+.article h2 {
+    @apply text-2xl font-bold;
+}
+
+.article h3 {
+    @apply text-xl font-bold;
+}
+
+.article img,
+.article pre {
+    @apply my-8;
+}
+
+.article img {
+    @apply shadow-lg;
+}
+
+.article p code {
+    @apply bg-green-light px-1 text-base;
+}
```

Comment:
I think we still need quite a few styling extra's here. Think blockquotes, h4-6, links.

It's best that you try to search for Markdown examples and try to render them. Like this one: http://www.unexpected-vortices.com/sw/rippledoc/quick-markdown-example.html